babel.config.js

```diff
@@ -1,3 +1,16 @@
 module.exports = {
   presets: ['module:metro-react-native-babel-preset'],
+  plugins: [
+    [
+      'module:react-native-dotenv',
+      {
+        moduleName: '@env',
+        path: '.env',
```

Comment:
Let's add this file to the `.gitignore` to avoid situations when someone will push it with their changes